build/pack.ps1

```diff
@@ -34,11 +34,31 @@ function Pack-One() {
     }
 }
 
+function Pack-Dotnet() {
+    Param($project, $option1="", $option2="", $option3="")
```

Comment:
You can use the `$Args` automatic variable to capture extra parameters:

```pwsh
PS> function Foo { param($x); Write-Host $Args; }
PS> Foo 3 4 -b 5
4 -b 5
```

Reply:
Sure, again, copy pasted (occasionally even I do that ;) -  for the sake of sticking with what is already there).